kong/plugins/rate-limiting/handler.lua

```diff
@@ -62,6 +62,14 @@ local function get_usage(conf, api_id, identifier, current_timestamp, limits)
   return usage, stop
 end
 
+local function get_api_id(conf)
+  if conf.cross_api then
+    return "00000000-0000-0000-0000-000000000000"
```

Comment:
defining this as a constant would make its meaning a bit clearer, rather than having magic strings floating around the codebase.